sendtweet

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+#########################################################################
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#########################################################################
+
+# You can also define these values in another file named .twitterkeys
+# by placing it in the same directory.
+
+CONSUMER_KEY='SPg4RX8Zdn5nxzP01IgCQGGQB'
```

Comment:
this should remain in an external file, which (thanks to gitignore) is not added to the public repository.